build/webpack.common.config.js

```diff
@@ -44,6 +46,19 @@ module.exports = language => {
                         name: '[name].[ext]',
                     },
                 },
+                // Transpile BUIE JS only if we are sym-linking
+                {
+                    test: /\.(js|mjs|ts|tsx)$/,
+                    include: /node_modules\/box-ui-elements/,
```

Comment:
It's a little unusual to use both `include` and `exclude` for the same dependency. Does this affect our production bundle at all? Is it needed for the change to work?

Reply:
I copied this rule from EUA. It doesn't look like the exclude is needed. I do think it may affect production bundle as it's a common webpack file. I will investigate more the implications on production